src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -13,6 +13,9 @@
 using namespace Microsoft::Console;
 using namespace Microsoft::Console::VirtualTerminal;
 
+// the console uses 0xffffffff as an "invalid color" value
+constexpr COLORREF INVALID_COLOR = 0xffffffff;

```

Comment:
if the console uses this as an invalid color, I really hope there's a constant for it somewhere else !

Reply:
oh this line originally for ResetCursorColor  and was accidentally removed in #7578. I didn’t write it. Just add it back.